java/arcs/android/sdk/host/IntentArcHostAdapter.kt

```diff
@@ -76,6 +76,8 @@ class IntentArcHostAdapter(
         sendIntentToHostServiceForResult(hostComponentName.createUnpauseArcHostIntent(hostId))
     }
 
+    override suspend fun waitForArcIdle(arcId: String) = throw NotImplementedError()
```

Comment:
You can use TODO("reason") which throws NotImplementedError, but shows up as TODO in audits in the IDE.